app/models/grda_warehouse/client_notes/base.rb

```diff
@@ -40,6 +40,14 @@ class Base < GrdaWarehouseBase
       where(type: types)
     end
 
+    scope :active, -> do
+      where.not(id: expired)
```

Comment:
This is going to add a sub-query, which probably won't matter for performance reasons with notes, which will usually be small, but maybe is worth cleaning up so we don't introduce the pattern.
```suggestion
      where(expiration_date: nil).or(where(expiration_date: Date.current + 1.days ..))
```

Reply:
cleaned up subquery [here](https://github.com/greenriver/hmis-warehouse/pull/5160/commits/7cce5e5a24f96e79d8341e700963bd0b8a6ff59f)